### DISCUSSION TITLE
Recreating onboarding branch

### DISCUSSION THREAD

**Discussion by codewode** [renovatebot/renovate#13562] [Request Help] [answered]
I merged my onboarding PR with the config defining "main" branch. If now I change the config with a different baseBranch (a feature branch), then it still loads the main branch. Therefore, I removed the config file in an anticipation that on next build the Renovate Ext will create a new onboarding branch, this never happend.

this is my config

```

{
  "platform": "azure",
  "rangeStrategy": "auto",
  "baseBranches": ["renovate-me"],
  "packageRules": [
    { 
      "matchBaseBranches": ["renovate-me"],
      "matchUpdateTypes": ["minor", "patch", "pin", "digest"]
    }
  ],
  "followTag": ["renovate"]
}
```

**Comment by rarkins** [comment, marked as answer]
https://docs.renovatebot.com/getting-started/installing-onboarding/#nuke-config-and-re-onboard

**Comment by codewode** [comment]
the document nuke-config-and-re-onboard states the following steps.

1. Find your original Configure Renovate PR
2. Rename the original PR to something else, e.g. Configure Renovate - old

i have no possibility to rename, since the branch is already merged. Did you mean, i should overwrite the commit history on main branch?

**Reply by rarkins** [reply]
If your platform (which you have not mentioned in the above) does not allow for renaming of merged PRs, then getting a new onboarding PR will not be possible. Platforms like GitHub and GitLab do allow for renaming, for example.
